dev-cpp/yaml_cpp/yaml_cpp-0.6.1.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="A YAML parser and emitter in C++"
+DESCRIPTION="yaml-cpp is a YAML parser and emitter in C++ matching the YAML \
+1.2 spec."
+HOMEPAGE="https://github.com/jbeder/yaml-cpp"
+COPYRIGHT="2018 Jesse Beder"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-$portVersion.tar.gz"
+CHECKSUM_SHA256="25ec37e6d82ab8c485926d69a5567741c7263515f8631e5dcb3fb4708e6b0d0d"
+SOURCE_DIR="yaml-cpp-yaml-cpp-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	yaml_cpp$secondaryArchSuffix = $portVersion compat >= 0.6
+	lib:libyaml_cpp$secondaryArchSuffix = $portVersion compat >= 0.6
+	lib:libgmock$secondaryArchSuffix
+	lib:libgmock_main$secondaryArchSuffix
```

Comment:
`libgmock` is a part of the bundled `gtest`, which we already packaged. You should get rid of them.

```diff
-	lib:libgmock$secondaryArchSuffix
-	lib:libgmock_main$secondaryArchSuffix